docs/csharp/whats-new/csharp-11.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What's new in C# 11 - C# Guide
+description: Get an overview of the new features coming in C# 11.
+ms.date: 03/04/2022
+---
+# What's new in C# 11
+
+Beginning with the .NET 6.0.200 SDK or Visual Studio 2022 version 17.1, preview features in C# are available for you to try.
+
+> [!IMPORTANT]
+> These are currently preview features. You must [set `<LangVersion>` to `preview`](../language-reference/compiler-options/language.md#langversion) to enable these features. Any feature may change before its final release. These features may not all be released in C# 11. Some may remain in a preview phase for longer based on feedback on the feature.
+
+The following features are available in the 6.0.200 version of the .NET SDK. They are available in Visual Studio 2022 version 17.1.
+
+- [Generic attributes](#generic-attributes).
+- [static abstract members in interfaces](#static-abstract-members-in-interfaces).
+
+You can download the latest .NET 6 SDK from the [.NET downloads page](https://dotnet.microsoft.com/download). You can also download [Visual Studio 2022](https://visualstudio.microsoft.com/vs/), which includes the .NET 6 SDK.
+
+## Generic attributes
+
+You can declare a [generic class](../programming-guide/generics/generic-classes.md) whose base class is <xref:System.Attribute?displayProperty=fullName>. This provides a more convenient syntax for attributes that require a <xref:System.Type?displayProperty=nameWithType> parameter. Previously, you'd need to create an attribute that takes a `Type` as its constructor parameter:
+
+```csharp
+public class TypeAttribute : Attribute
+{
+   public TypeAttribute(Type t) => ParamType = t;
+
+   public Type ParamType { get; }
+}
+```
+
+And to apply the attribute, you use the [`typeof`](../language-reference/operators/type-testing-and-cast.md#typeof-operator) operator:
+
+```csharp
+[TypeAttribute(typeof(string))]
+public string Method() => default;
+```
+
+Using this new feature, you can create a generic attribute instead:
+
+```csharp
+public class GenericAttribute<T> : Attribute { }
+```
+
+Then, specify the type parameter to use the attribute:
+
+```csharp
+[GenericAttribute<string>()]
+public string Method() => default;
+```
+
+You can apply a fully closed constructed generic attribute. In other words, all type parameters must be specified. For example, the following is not allowed:
```

Comment:
I think we need a link here to the ideas of closed and constructed. I thought they meant the same thing. And/or a sentence here. 

As I think of this as a constructed type, the comment below might should say "fully constructed". But I also could be missing a nuance